app/src/main/res/navigation/nav_graph.xml

```diff
@@ -49,6 +49,8 @@
 
         <action
             android:id="@+id/action_from_list_to_edit"
-            app:destination="@id/EditFragment" />
+            app:destination="@id/EditFragment"
+            app:enterAnim="@anim/slide_in_right"
```

Comment:
`app:exitAnim` would be great

Reply:
Can currently not be implemented, since the navigation only goes one way